modin/core/storage_formats/pandas/utils.py

```diff
@@ -38,40 +40,48 @@ def get_default_chunksize(length, num_splits):
     )
 
 
-def compute_chunksize(df_shape, num_splits, default_block_size=32, axis=None):
+def compute_chunksize(
```

Comment:
Given how this function is now written and used, I just wonder if we should change the logic to just accept _one_ count (by one axis), and in the case when we need two splits (both rows and columns) just call it twice...

I think it would make code simpler.

Reply:
I thought about it, if you think the same, then it is definitely worth a try :)

Reply:
@vnlitvinov ready for review